### PR TITLE
Upgrade Kaptain for Konvoy 1.x

### PR DESCRIPTION
**Pull request by gvasquezvargas**
The topic now includes information on how to update Kaptain on Konvoy 1.x and not only on Konvoy 2.x.
Some restructuring of the article was done as well, any feedback is appreciated.
Jira: https://jira.d2iq.com/browse/COPS-7176?filter=57924

### CODE REVIEW

File: pages/dkp/kaptain/1.3.0/upgrade/index.md [hunk 1]
```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
...
       - configRepository: https://github.com/mesosphere/kubeaddons-kaptain
         configVersion: stable-1.20-1.4.0
         addonsList:
           - name: knative
             enabled: true
   ```
+  
+## Upgrade Kaptain 
+
+1. Perform the upgrade:
```
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/upgrade/index.md:48]:
```suggestion
Run the upgrade according to your Konvoy version:
```

File: pages/dkp/kaptain/1.3.0/upgrade/index.md [hunk 2]
```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
...
+
+### For Konvoy 1.x: 
+
+Add the following properties to your `params.yaml` file:
```
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/upgrade/index.md:52]:
```suggestion
1.   Add these properties to your `params.yaml` file:
```

File: pages/dkp/kaptain/1.3.0/upgrade/index.md [hunk 3]
```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
...
+### For Konvoy 1.x: 
+
+Add the following properties to your `params.yaml` file: 
+
+  ```bash
+  dkpPlatformVersion: "1" 
+  installMinioOperator: "true"
+  ```
+
+And upgrade Kaptain:
```
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/upgrade/index.md:59]:
```suggestion
1.   And the Kaptain upgrade : 
```

File: pages/dkp/kaptain/1.3.0/upgrade/index.md [hunk 4]
```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
...
+
+And upgrade Kaptain: 
+
+  ```bash
+  kubectl kudo upgrade --instance kaptain --namespace kubeflow ./kubeflow-1.4.0_1.3.0.tgz -P params.yaml
+  ```
+  
+### For Konvoy 2.x:
+
+Upgrade Kaptain:
```
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/upgrade/index.md:67]:
```suggestion
1.  Upgrade Kaptain:
```

File: pages/dkp/kaptain/1.3.0/upgrade/index.md [hunk 5]
```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
...
 
-* [Download][download] the `kubeflow-1.4.0_1.3.0.tgz` tarball from the support website.
-* Upgrade Kaptain:
   ```bash
   kubectl kudo upgrade --instance kaptain --namespace kubeflow ./kubeflow-1.4.0_1.3.0.tgz
   ```
-* Monitor the upgrade process by running:
+
+2. Monitor the upgrade process by running:
```
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/upgrade/index.md:73]:
```suggestion
1.  Monitor the upgrade process by running:
```

Comment (Colin1841) [reply pages/dkp/kaptain/1.3.0/upgrade/index.md:73]: Markdown dictates DO NOT use sequential numbers, always use "1.  " (note the two blank spaces)

File: pages/dkp/kaptain/1.3.0/upgrade/index.md [hunk 6]
```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
...
+  
   ```bash
   kubectl kudo plan status --instance kaptain --namespace kubeflow
   ```
 
-After the upgrade completes, log in to Kaptain:
+3. After the upgrade completes, log in to Kaptain.
 
-* Discover the cluster endpoint and copy it to the clipboard.
+1. Discover the cluster endpoint and copy it to the clipboard.
```
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/upgrade/index.md:81]:
```suggestion
1.  Locate the cluster endpoint and copy it to your clipboard.
```

File: pages/dkp/kaptain/1.3.0/upgrade/index.md [hunk 7]
```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
...
+  
   If you are running Kaptain _on-premises_, use this command:
+
   ```bash
   kf_uri=$(kubectl get svc kubeflow-ingressgateway --namespace kubeflow -o jsonpath="{.status.loadBalancer.ingress[*].ip}") && echo "https://${kf_uri}"
   ```
-  Or if you are running Kaptain on _AWS_, use this command:
+
+  If you are running Kaptain on _AWS_, use this command:
```
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/upgrade/index.md:89]:
```suggestion
  **OR** If you are running Kaptain on _AWS_, use this command:
```

File: pages/dkp/kaptain/1.3.0/upgrade/index.md [hunk 8]
```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
...
+  
   ```bash
   kf_uri=$(kubectl get svc kubeflow-ingressgateway --namespace kubeflow -o jsonpath="{.status.loadBalancer.ingress[*].hostname}") && echo "https://${kf_uri}"
   ```
-* Get the login credentials from Konvoy to authenticate:
+
+5. Get the login credentials from Konvoy to authenticate:
```
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/upgrade/index.md:95]:
```suggestion
1.  Obtain the login credentials from Konvoy to authenticate:
```

**Review by Colin1841 — APPROVED** [review]

**Review by alexchrisramos — APPROVED** [review]
Looks good for me!

**Review by cneth — APPROVED** [review]

**Pull request merged**